pkg/steps/template_test.go

```diff
@@ -230,11 +233,81 @@ func TestOperateOnTemplatePods(t *testing.T) {
 				},
 			},
 		},
+		{
+			testID: "resources defined in the config but the test container has existing resources, no changes expected",
```

Comment:
I think we also want to test the other scenario, where the resources are propagated from the config to the container.

Reply:
addressed